### PR TITLE
move `VALID_INSTRUCTIONS` to instruction file

### PR DESCRIPTION
**Pull request by tcoratger**

### CODE REVIEW

File: crates/brainfuck_vm/src/instruction.rs [hunk 1]
```diff
@@ -59,6 +60,39 @@ impl FromStr for InstructionType {
...
+        }
+    }
+
+    /// Convert an [`InstructionType`] to a [`BaseField`]
+    pub const fn to_base_field(&self) -> BaseField {
+        BaseField::from_u32_unchecked(self.to_u32())
+    }
+}
+
+// Define all valid instructions as [`BaseField`] values
```
Comment (zmalatrax) [inline crates/brainfuck_vm/src/instruction.rs:84]: missing one '/' for header doc

File: crates/brainfuck_vm/src/instruction.rs [hunk 2]
```diff
@@ -59,6 +60,39 @@ impl FromStr for InstructionType {
...
+pub const VALID_INSTRUCTIONS: [BaseField; 8] = [
```
Comment (zmalatrax) [inline crates/brainfuck_vm/src/instruction.rs:85]: Should we call it `VALID_INSTRUCTIONS_BF` as these are valid instructions as basefield?

Comment (tcoratger) [reply crates/brainfuck_vm/src/instruction.rs]: yes let's go, just fixed

File: crates/brainfuck_vm/src/instruction.rs (hunk 1 above)
Comment (tcoratger) [reply crates/brainfuck_vm/src/instruction.rs]: thanks fixed

**Review by zmalatrax — APPROVED** [review]

**Pull request merged**